src/clusterfuzz/_internal/tests/core/bot/tasks/utasks/minimize_task_test.py

```diff
@@ -270,6 +270,86 @@ def test_null_crash_result_raises(self):
       minimize_task._extract_crash_result(crash_result, command)  # pylint: disable=protected-access
 
 
+@test_utils.with_cloud_emulators('datastore')
+class UTaskPostprocessTest(unittest.TestCase):
+  """Tests for utask_postprocess."""
+
+  def setUp(self):
+    helpers.patch_environ(self)
+    helpers.patch(self, [
+        'clusterfuzz._internal.bot.tasks.utasks.minimize_task.finalize_testcase',
+    ])
+
+  def _get_generic_input(self):
+    testcase = data_types.Testcase()
+    testcase.put()
+    uworker_input = uworker_io.UworkerInput(
+        job_type='job_type', testcase_id='testcase_id', testcase=testcase)
+    uworker_input = uworker_io.serialize_uworker_input(uworker_input)
+    uworker_input = uworker_io.deserialize_uworker_input(uworker_input)
+    return uworker_input
+
+  def _create_output(self, uworker_input=None, **kwargs):
+    uworker_output = uworker_io.UworkerOutput(**kwargs)
+    uworker_output = uworker_io.serialize_uworker_output(uworker_output)
+    uworker_output = uworker_io.deserialize_uworker_output(uworker_output)
+    if uworker_input:
+      uworker_output.uworker_input = uworker_input
+    return uworker_output
+
+  def test_error_does_not_finalize_testcase(self):
+    """Check that an output with an error does not finalize a testcase."""
+    uworker_output = self._create_output(
+        error=uworker_msg_pb2.ErrorType.UNHANDLED)
+    minimize_task.utask_postprocess(uworker_output)
+    self.assertFalse(self.mock.finalize_testcase.called)
+
+  def test_missing_uworker_input_raises(self):
+    """Check that an output missing the uworker_input field raises."""
```

Comment:
Can you change function name to `raises_error` and the comment to "field raises an error."
Also, another annoying nit, but comments are supposed to be "descriptive style" according to the style guide. So "checks" instead of "check".

Reply:
oof def did not read the style guide oops - done thank you!!

Reply:
Didn't expect you to! no worries!

Reply:
We're not super consistent about this one